### ISSUE TITLE
Quotes and tags are not rendered when exported to Android

### ISSUE THREAD

**Issue by albertad23** [tolgee/tolgee-platform#2188]
Hi,

We have recognized that the encoding `\"` on iOS doesn't seem correct.

For example, if I have a string:
`\"halo\"`

The export for iOS would look like:
`"testString" = "\\\"halo\\\"";`

However, for Android, the export appears correct and as expected:
`<string name="testString">\"halo\"</string>`

Somehow, an extra `\\` is added on iOS export.
Could someone please take a look?

Thanks!

**Comment by JanCizmar** [comment]
Hello! Thanks for reporting! I am looking into this.

**Comment by JanCizmar** [comment]
The `"` and `\` characters has to be escaped in iOS, what is they expected behaviour you propose? If we exported the string as `"testString" = "\"halo\"";` the actual value rendered in your Apple app would be `"halo"`, which is not what you want, right?

**Comment by albertad23** [comment]
The expected behavior for me is that the export for iOS and Android should be the same.

If I have a string: 
`"testString" = "\"halo\"";`

the expected result for iOS would be: 
`"testString" = \"halo\";`

 which is the same with current export for Android.

**Comment by JanCizmar** [comment]
If you need to render a `\halo\` your string in Tolgee Platfrom should be equal to this. Tolgee correctly exports such key to `strings` files as `"mobile_key" = "\\halo\\";` and to Android as  `<string name="mobile_key">\halo\</string>`. 

This seems perfectly correct exporting behaviour.

**Comment by albertad23** [comment]
I think I need to rephrase my questions:

We aim to generate export strings for iOS and Android from a single input string.

The expected export string for both iOS and Android should be:
`"Lorem ipsum dolar is the \"best"\ now";`

When I input this string into Tolgee:
`"Lorem ipsum dolar is the \"best\" now";`

Tolgee provides the following export:
iOS: `"testString" = "Lorem ipsum dolar is the \\\"best\\\" now";`
Android: `<string name="testString">Lorem ipsum dolar is the \"best\" now</string>`

The result for Android is correct, but for iOS, it contains too many backslashes at the beginning and end of “best”.

However, if I input this string:
`"Lorem ipsum dolar is the "best" now";`

Tolgee exports:
iOS: `"testString" = Lorem ipsum dolar is the \"best\" now";`
Android: `<string name="testString">Lorem ipsum dolar is the "best" now</string>`
For iOS, this is the expected string, but for Android, the exported string is incorrect.

In summary:
We need a string that generates the correct export for both iOS and Android, namely: `"Lorem ipsum dolar is the \"best\" now";`

**Comment by JanCizmar** [comment]
Oh, I see. Android just removes the quotes when rendering the string. This means we probably have to reconsider how we export and import android files. The ignored data should be ignored when importing to Tolgee as well, so we know it was escaped correctly when importing. I will look into it tomorrow. Thanks for reporting and explanation!

**Comment by JanCizmar** [comment]
It's deployed on Tolgee Cloud now. 🚀 Here's the docs: https://tolgee.io/platform/formats/android_xml#strings-wrapped-with-cdata
